### ISSUE TITLE
update pre-trained checkpoint in readme

### ISSUE THREAD

**Issue by freelw** [freelw/cpp-transformer#39]
update pre-trained checkpoint in readme

**Comment by freelw** [comment]
Since we have removed the gradients from the checkpoint, the previous file format has become invalid.
